packages/network/package.json

```diff
@@ -34,66 +34,40 @@
     "test": "tsc --noEmit && jest --forceExit"
   },
   "dependencies": {
-    "@latticexyz/common": "workspace:*",
-    "debug": "^4.3.4",
-    "lodash": "^4.17.21",
-    "viem": "^0.3.18"
-  },
-  "devDependencies": {
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
     "@improbable-eng/grpc-web": "^0.15.0",
+    "@latticexyz/common": "workspace:*",
     "@latticexyz/recs": "workspace:*",
     "@latticexyz/schema-type": "workspace:*",
     "@latticexyz/services": "workspace:*",
     "@latticexyz/solecs": "workspace:*",
     "@latticexyz/store": "workspace:*",
     "@latticexyz/utils": "workspace:*",
-    "@protobuf-ts/grpcweb-transport": "^2.7.0",
```

Comment:
I couldn't find any usages of this and the example app seems to run locally without issue.

Reply:
yeah we just use the web transport exports of `@improbable-eng/grpc-web` (which is installed by `nice-grpc-web`)